src/pages/iou/ModalHeader.js

```diff
@@ -32,25 +33,33 @@ const ModalHeader = (props) => (
         <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.flexGrow1, styles.justifyContentBetween, styles.overflowHidden]}>
             {props.shouldShowBackButton && (
                 <Tooltip text={props.translate('common.back')}>
-                    <TouchableOpacity
+                    <PressableWithFeedback
                         onPress={props.onBackButtonPress}
                         style={[styles.touchableButtonImage]}
+                        accessibilityRole="button"
+                        accessibilityLabel={props.translate('common.back')}
+                        // disable hover dim for switch
```

Comment:
Out of curiosity, why do we need to do this here and below?

Reply:
I forgot to change comments from the other component 😅 But we are adding 
```
hoverDimmingValue={1}
pressDimmingValue={0.2}
```
to make the `PressableWithFeedback` behave like the `TouchableOppacity` component. By default `PressableWithFeedback` change its opacity on hover, so we want to disable that by setting it to 1